src/main/java/frc/robot/Constants.java

```diff
@@ -27,6 +27,17 @@ public static class ArmConstants {
         public static final int SHOULDER_FOLLOWER_ID = 30;
         public static final int ELBOW_ID = 32;
         public static final int INTAKE_ID = 31;
+        public static final double STATE_EPSILON = 10;
```

Comment:
This constant has to be tuned before we merge.

Reply:
Change the naming to `STATE_TOLERANCE` Make it readable. While epsilon could also be used it is less consistent with the naming conventions of the arm.

Reply:
Sure, I chose that name because it is consistent with code from previous years, but if you want to change it that's fine.